worker/paidAction.js

```diff
@@ -328,11 +314,31 @@ export async function paidActionForwarded ({ data: { invoiceId, withdrawal, ...a
     },
     ...args
   }, { models, lnd, boss })
+
+  if (transitionedInvoice) {
```

Comment:
I moved this because

1. if it errors, it won't block the transition
2. before, the logging might've succeeded even if the transition failed

... likewise with the other transitions